source/common/http/conn_manager_impl.cc

```diff
@@ -765,6 +765,7 @@ void ConnectionManagerImpl::ActiveStream::decodeHeaders(ActiveStreamDecoderFilte
     const auto current_filter_end_stream =
         decoding_headers_only_ || (end_stream && continue_data_entry == decoder_filters_.end());
     FilterHeadersStatus status = (*entry)->decodeHeaders(headers, current_filter_end_stream);
+    (*entry)->end_stream_ = current_filter_end_stream;
```

Comment:
Is it possible to remove `current_filter_end_stream` variable and just set the `end_stream_` variable directly and us that for the call? I'm not sure if setting before call will break anything but it seems cleaner and easier to understand if so. same elsewhere.

Reply:
Done